app.js

```diff
@@ -120,6 +123,96 @@ app.get('/api/nodes', function(req, res) {
   });
 });
 
+app.get('/api/accounts', function(req, res) {
+  var day = moment();
+  var response = [];
+
+  var multi = redisClient.multi();
+
+  var days = 30;
+
+  for (var i = 0; i < days; i++) {
+    response.push({date: day.format("DD-MM")});
+    multi.get('accounts_'+day.format("YYYY-MM-DD"));
+    day = day.subtract(1, 'days')
+  }
+
+  multi.exec(function (err, redisRes) {
+    if (err) {
+      res.sendStatus(500);
+      console.error(err);
+      res.error("Error");
+      return;
+    }
```

Comment:
We use very similar code in 3 places now. Maybe let's extract it to a new function?

Reply:
yep I extracted this.